store/sqlstore/upgrade.go

```diff
@@ -672,12 +671,12 @@ func UpgradeDatabaseToVersion511(sqlStore SqlStore) {
 		// Enforce all teams have an InviteID set
 		var teams []*model.Team
 		if _, err := sqlStore.GetReplica().Select(&teams, "SELECT * FROM Teams WHERE InviteId = ''"); err != nil {
-			mlog.Error("Error fetching Teams without InviteID: " + err.Error())
+			mlog.Error("Error fetching Teams without InviteID:", mlog.Err(err))
 		} else {
 			for _, team := range teams {
 				team.InviteId = model.NewId()
 				if _, err := sqlStore.Team().Update(team); err != nil {
-					mlog.Error("Error updating Team InviteIDs: " + err.Error())
```

Comment:
Same here, what about adding a `team` or `team_id` parameter to this? Also we can remove the `:`.

Reply:
done